pkg/whip/whip_proxy_handler.go

```diff
@@ -313,11 +323,35 @@ func (h *proxyWhipHandler) ICERestartWHIPResource(ctx context.Context, req *rpc.
 }
 
 func (h *proxyWhipHandler) WHIPRTCConnectionNotify(ctx context.Context, req *rpc.WHIPRTCConnectionNotifyRequest) (*google_protobuf2.Empty, error) {
-	h.logger.Infow("WHIPRTCConnectionNotify", "participantID", h.participantID, "req", req)
+	tctx, done := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
```suggestion
	tctx, done := context.WithTimeout(ctx, 10*time.Second)
```
to bind to the request lifetime and cancel work if the RPC is canceled?

Reply:
makes sense - it all depends on the semantics of canceling the RPC - if we still want to update it even though it's canceled and we don't need to return as fast as possible